DirectGuiExtension/DirectSplitFrame.py

```diff
@@ -98,26 +98,31 @@ def __init__(self, parent = None, **kw):
             text_scale = 0.05,
             text_align = TextNode.ACenter,
             state = 'normal')
+
+        # Call option initialization functions
+        self.initialiseoptions(DirectSplitFrame)
+
         self.splitter.bind(DGG.ENTER, self.enter)
         self.splitter.bind(DGG.EXIT, self.exit)
         self.splitter.bind(DGG.B1PRESS, self.dragStart)
         self.splitter.bind(DGG.B1RELEASE, self.dragStop)
 
-        self.setSplitter(self['splitterWidth'])
+        self.setSplitter()
 
         self.skipInitRefresh = False
         # initialize once at the end
         self.refresh()
 
-    def setSplitter(self, width=0.02):
+    def setSplitter(self):
         if not hasattr(self, "splitter"): return
         if self['showSplitter'] == False:
             self.splitter.hide()
+            self._splitterWidth = self["splitterWidth"]
             self['splitterWidth'] = 0
         else:
             self.splitter.show()
-            self['splitterWidth'] = width
-
+            if self["splitterWidth"] == 0:
```

Comment:
Why do you allow splitter width updates only if the width is zero? It should also be possible otherwise.

Reply:
Ah yes, you're right. Makes sense to have it that way.